ios/browser/api/ads/brave_ads.mm

```diff
@@ -248,10 +261,26 @@ - (void)initializeWithSysInfo:(BraveAdsSysInfo*)sysInfo
                                  : brave_ads::mojom::BuildChannelInfo::New();
   ads->SetBuildChannel(std::move(cppBuildChannelInfo));
   ads->SetFlags(brave_ads::BuildFlags());
+
+  if (walletInfo && walletInfo.cppObjPtr) {
+    ads->OnRewardsWalletDidChange(walletInfo.cppObjPtr->payment_id,
+                                  walletInfo.cppObjPtr->recovery_seed);
```

Comment:
`cppObjPtr` will be make a new one each time so it wouldn't be enough to check it once. Best to assign it to a variable first then check and use that.  Otherwise you can also instead do `base::SysNSStringToUTF8(walletInfo.paymentId)` instead

Reply:
Fixed